### PR TITLE
Feature/endpoints

### PR DESCRIPTION
**Pull request by jitendra2329**
Endponits, Put and Delete implemented.

### CODE REVIEW

File: app/service/DeliveriesService.scala [hunk 1]
```diff
@@ -1,57 +1,68 @@
...
-        firstName = Some("Bhavya")
-      )
-    )
-  )
-  override def getByid(merchantId: String, id: String): Either[Seq[String], Delivery] = {
-    db.get(id) match {
-      case Some(delivery) => Right(delivery)
-      case None => Left(Seq("Delivery Not Found"))
+//
+//  private var database: Map[String, Delivery] = Map(
```
Comment (Bhavya-verma-knoldus) [inline app/service/DeliveriesService.scala]: Let's remove this commented code if not needed.

Comment (jitendra2329) [reply app/service/DeliveriesService.scala]: okay, will to do it.

Comment (jitendra2329) [reply app/service/DeliveriesService.scala]: *do it.

**Review by Bhavya-verma-knoldus — APPROVED** [review]

**Comment by jitendra2329** [comment]
Should I merge the PR now?

**Comment by Bhavya-verma-knoldus** [comment]
> Should I merge the PR now?

Yes

**Pull request merged**